docs/source/api_reference/forecasting.rst

```diff
@@ -146,6 +146,18 @@ Direct and recursive - 3rd party
     DartsXGBModel
 
 
+Bias Adjusted Forecasters
```

Comment:
Can I suggest to move this after "prediction intervals", and call it "Calibration and bias adjustment", anticipating things like probabilistic calibration? As those require probabilistic forecasters (such as the variance), it makes sense to have it directly after.